internal/initwd/from_module_test.go

```diff
@@ -281,6 +282,7 @@ func TestDirFromModule_rel_submodules(t *testing.T) {
 	}
 	t.Cleanup(func() {
 		os.Chdir(oldDir)
+		runtime.GC()
```

Comment:
Can you explain / add a comment why we need manual GC here?

Reply:
Sure, there are many TempDir RemoveAll cleanup errors that are coming up when running tests on Windows. This is because a file is being written to or created without properly being closed. As a result, TempDir cannot delete the temp files on Windows because they are blocked by another process. `t.cleanup()` is run when the test completes so it doesn't not affect the logic inside the test. By manually calling `runtime.GC()`, it triggers finalisers to run. If a file has no more references and a finaliser is set to close the file then `runtime.GC()` will cause this file handle, that is no longer being used, to close. Then the temporary file in Windows can be deleted.

Reply:
Thanks!